src/App.tsx

```diff
@@ -1,29 +1,35 @@
 import React from 'react';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Header from './components/Header';
 import logo from './logo.svg';
 import { FirebaseProvider } from './components/Firebase';
 import { firebaseConfig } from './constants';
 import './App.css';
 
-const App: React.FC = () => {
+const App: React.FC = (): JSX.Element => {
```

Comment:
Can we remove the `JSX.Element`? Typescript should be able to infer from `React.FC`.

Reply:
fb1ceec